daprdocs/content/en/developing-applications/building-blocks/actors/howto-actors.md

```diff
@@ -315,24 +315,10 @@ Since version 1.3, applications can now enable partitioning of actor reminders i
 If the number of partitions is not enough, it can be changed and Dapr's sidecar will automatically redistribute the reminders's set.
 
 ### Enabling actor reminders partitioning
-Actor reminders partitioning is currently in preview, so enabling it is a two step process.
-
-#### Preview feature configuration
-Before using reminders partitioning, actor type metadata must be enabled in Dapr. For more information on preview configurations, see [the full guide on opting into preview features in Dapr]({{< ref preview-features.md >}}). Below is an example of the configuration:
-
-```yaml
-apiVersion: dapr.io/v1alpha1
-kind: Configuration
-metadata:
-  name: myconfig
-spec:
-  features:
-    - name: Actor.TypeMetadata
-      enabled: true
-```
+Actor reminders partitioning is a fully supported feature since Dapr version `1.8`.
```

Comment:
It is not needed to keep a version in the docs, since one day this just becomes old. Best to remove these.
```suggestion
```